packages/framework/tests/Unit/Component/Cron/CronFacadeTest.php

```diff
@@ -118,7 +119,9 @@ private function createCronFacade(CronConfig $cronConfig, CronModuleFacade $cron
         /* @var $mailerMock \Shopsys\FrameworkBundle\Model\Mail\Mailer */
         $mailerMock = $this->createMock(Mailer::class);
 
-        return new CronFacade($loggerMock, $cronConfig, $cronModuleFacade, $mailerMock);
+        $cronModuleExecutor = new CronModuleExecutor(240);
```

Comment:
It seems like an unnecessarily long timeout for use in unit tests. I would guess even 1 second should be enough. But I'm OK with 2 minutes as well...

Reply:
It seems like it but I don't think that we should change it in this PR

Reply:
OK